Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -254,6 +255,7 @@ protected theorem prod_snd : LipschitzWith 1 (@Prod.snd α β) :=
   LipschitzWith.of_edist_le fun _ _ => le_max_right _ _
 #align lipschitz_with.prod_snd LipschitzWith.prod_snd
 
+/-- The product of Lipschitz functions is Lipschitz. -/
```

Comment:
Do you want to reword this to make it clear that it's not the functions multiplied together? I'm not sure of a good wording, but give it a little thought before merging.

Thanks!

bors d+

Reply:
Maybe "A function to a product type created from two Lipschitz functions is itself Lipschitz"? Not ideal, but the best I could come up with...

Reply:
Reworded, taking both your suggestions into account. An improvement indeed!